### PR TITLE
SIGNL4 - Mobile Alerting added

### PR DESCRIPTION
**Pull request by rons4**
This should replace the existing SIGNL4 connector.

---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [ ] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [ ] I attest that the connector works and I verified by deploying and testing all the operations. 
- [ ] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [ ] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [ ] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [ ] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [x] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [x] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [x] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

File: certified-connectors/SIGNL4 - Mobile Alerting/apiProperties.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+            "tabIndex": 2,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#000000",
+    "capabilities": [],
+    "publisher": "Ronald Czachara"
```
Comment (Amjed-Ayoub) [inline certified-connectors/SIGNL4 - Mobile Alerting/apiProperties.json]: @rons4 Please add the stackOwner property. Please see out template [here](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/apiProperties.json). 
```
    "stackOwner": "Name of the owner of the service or product."
```

File: certified-connectors/SIGNL4 - Mobile Alerting/README.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# SIGNL4: Mobile Alerting and Incident Response
```
Comment (Amjed-Ayoub) [inline certified-connectors/SIGNL4 - Mobile Alerting/README.md:1]: @rons4 Please include all required sections in the readme file. please see out template [here ](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md)and for raw code view [here](https://[raw.githubusercontent.com/microsoft/PowerPlatformConnectors/dev/templates/certified-connectors/readme.md](https://raw.githubusercontent.com/microsoft/PowerPlatformConnectors/dev/templates/certified-connectors/readme.md)).

File: certified-connectors/SIGNL4 - Mobile Alerting/apiDefinition.swagger.json [hunk 3]
```diff
@@ -0,0 +1,15101 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "contact": {
+      "name": "Derdack GmbH",
+      "url": "https://www.signl4.com/feedback-we-love-hear-from-you/",
+      "email": "success@signl4.com"
+    },
+    "description": "SIGNL4 offers critical mobile alerting, incident response and service dispatching for operating critical infrastructure. It alerts you persistently via app push, SMS text and voice calls including tracking, escalation, collaboration and duty planning.",
+    "license": {
```
Comment (Amjed-Ayoub) [inline certified-connectors/SIGNL4 - Mobile Alerting/apiDefinition.swagger.json]: @rons4 the license section can be removed.

File: certified-connectors/SIGNL4 - Mobile Alerting/apiDefinition.swagger.json [hunk 4]
```diff
@@ -0,0 +1,15101 @@
...
+      "name": "Derdack GmbH",
+      "url": "https://www.signl4.com"
+    },
+    "title": "SIGNL4 - Mobile Alerting",
+    "version": "2"
```
Comment (Amjed-Ayoub) [inline certified-connectors/SIGNL4 - Mobile Alerting/apiDefinition.swagger.json]: @rons4 please change the version to "2.0"

**Review by Amjed-Ayoub — CHANGES_REQUESTED** [review]

File: certified-connectors/SIGNL4 - Mobile Alerting/apiProperties.json (hunk 1 above)
Comment (rons4) [reply certified-connectors/SIGNL4 - Mobile Alerting/apiProperties.json]: Done.

File: certified-connectors/SIGNL4 - Mobile Alerting/apiDefinition.swagger.json (hunk 3 above)
Comment (rons4) [reply certified-connectors/SIGNL4 - Mobile Alerting/apiDefinition.swagger.json]: Done.

File: certified-connectors/SIGNL4 - Mobile Alerting/apiDefinition.swagger.json (hunk 4 above)
Comment (rons4) [reply certified-connectors/SIGNL4 - Mobile Alerting/apiDefinition.swagger.json]: Done.

File: certified-connectors/SIGNL4 - Mobile Alerting/README.md (hunk 2 above)
Comment (rons4) [reply certified-connectors/SIGNL4 - Mobile Alerting/README.md:1]: Revised.
Thanks for your quick help and for your comments, @Amjed-Ayoub.

**Review by Amjed-Ayoub — APPROVED** [review]
Hello @rons4 ,
I hope you are doing well.
Congratulations, your pull request is approved and merged. Please [submit the files to ISV Studio](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) or [update your submission](https://learn.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#update-submission-prior-to-certification) using your latest commit ID, please allow us up to 1-2 weeks to review your submission, and our Engineers will notify you in the “Activity Control” area in ISV Studio. Please [make sure that the icon meets all the requirements](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#design-an-icon-for-your-connector), that it’s size is 230x230, icon background is not white or transparent and it matches the iconBrandColor property in apiProperties.json file.
After making the submission in ISV Studio, please attach an intro.md file in the activity control section. This file is required and it's different from readme.md. You can [find an example of intro.md file on this page](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) at step 6.
Please [Create an environment in the Preview region](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#create-an-environment-in-the-preview-region). You will use this environment later to [test your connector](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#test-your-connector) after Microsoft is done with the functional verification for your connector. 
We expect all tests to be completed within 1-2 weeks.
If your connector passes all tests, the deployment process begins, and it typically takes up to 3 to 4 weeks to deploy your connector to all of our regions.
Please let me know if you have any questions or run into any problems while creating or updating your submission.
Thank you very much for working with us.

**Comment by rons4** [comment]
Thanks a lot @Amjed-Ayoub and I just submitted the files in ISV Studio.

**Pull request merged**